[WNMGDS-570] Update installation, css, font image documentation (#763)
CMSgov/design-system: packages/design-system-docs/src/pages/startup/installation.md

---
title: Installation
weight: 0
---

> How you implement the design system depends on the needs of your project and your workstyle. We highly recommend implementing the design system with `npm`, as allows you to easily update the design system version, but we also provide a direct download if `npm` will not work for you or your project.

Reviewer: ```suggestion
We highly recommend implementing the design system with `npm`, as it allows 
```

Reviewer: I think adding this `it` is still needed

- Typescript definition files
- Fonts and image

### Package file structure

The `dist` folder contains all the necessary files in the design system, your project should not be importing anything from `src`.

```
└── dist
    ├── components/
    │   └── index.js        Compiled JS entry point (CommmonJS)
    ├── css/
    │   └── index.css       Compiled CSS entry point
    ├── esnext/
    │   └── index.esm.js    Compiled JS entry point (ES Module)
    ├── fonts/
    ├── images/
    ├── scss/
    │   ├── base/           Base styles, HTML element selectors
    │   ├── components/     Component styles
    │   ├── settings/       Variables, mixins, and functions
    │   ├── utilities/      Utility classes for individual CSS properties
    │   └── index.scss      Precompiled SCSS entry point
    └── types/              Typescript definition files
```

## Download zip

<a href="{{root}}/download.zip">Download the zip file</a> from the latest release and extract the zip file to see the following folder structure. Like the `npm` package, the contents of `download/design-system/dist` contain design system code and assets.

```
└── download

>     ├── design-assets/      Design system assets (Sketch files, designer assets)

Reviewer: ```suggestion
    ├── design-assets/      Design system assets (Sketch files, fonts, designer assets)
```
